modules/phase_field/src/action/PolycrystalKernelAction.C

```diff
@@ -14,10 +14,12 @@ template<>
 InputParameters validParams<PolycrystalKernelAction>()
 {
   InputParameters params = validParams<Action>();
-  params.addRequiredParam<unsigned int>("op_num", "specifies the number of grains to create");
+  params.addRequiredParam<unsigned int>("op_num", "specifies the total number of grains (deformed + recrystallized) to create");
   params.addRequiredParam<std::string>("var_name_base", "specifies the base name of the variables");
   params.addParam<VariableName>("c", "Name of coupled concentration variable");
   params.addParam<Real>("en_ratio", 1.0, "Ratio of surface to GB energy");
+  params.addParam<unsigned int>("ndef", 0, "specifies the number of deformed grains to create");
+  params.addParam<bool>("deformation", false, "a flag to indicate Whether to account for deformation or not");
```

Comment:
It seems to me that the deformation parameter is redundant. `deformation = (ndef != 0)`